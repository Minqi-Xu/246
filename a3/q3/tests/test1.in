add body
add header
value CS246
parent
add text
value Hello
id 911
parent
add text2
id 911
value HelloNo.2
parent
add text3
id 911
value HelloWorld
parent
findId 911
up
findId 911
print
