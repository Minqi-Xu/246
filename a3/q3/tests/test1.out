[Command?]
<body> is now the current tag.
[Command?]
<header> is now the current tag.
[Command?]
<header> value set to 'CS246'.
[Command?]
<body> is now the current tag.
[Command?]
<text> is now the current tag.
[Command?]
<text> value set to 'Hello'.
[Command?]
<text> id set to '911'.
[Command?]
<body> is now the current tag.
[Command?]
<text2> is now the current tag.
[Command?]
<text2> id set to '911'.
[Command?]
<text2> value set to 'HelloNo.2'.
[Command?]
<body> is now the current tag.
[Command?]
<text3> is now the current tag.
[Command?]
<text3> id set to '911'.
[Command?]
<text3> value set to 'HelloWorld'.
[Command?]
<body> is now the current tag.
[Command?]
<text> is now the current tag.
[Command?]
<document> is now the current tag.
[Command?]
[Command?]
<document>
  <body>
    <header value='CS246'>
    </header>
    <text id='911' value='Hello'>
    </text>
    <text2 id='911' value='HelloNo.2'>
    </text2>
    <text3 id='911' value='HelloWorld'>
    </text3>
  </body>
</document>
[Command?]
